python/comps.py

```diff
@@ -0,0 +1,328 @@
+import numpy as np
+import numba as nb
+import pandas as pd
+
+# The number of bins to create for when binning data by sale price; e.g. if
+# this value is 10, the data will be binned into deciles
+NUM_PRICE_BINS = 30
```

Comment:
In https://github.com/ccao-data/model-res-avm/pull/182, I'll tweak this variable to make it configurable by the workflow runner.